commit 8593a9e1e4e8dd1f5dfac9b1304a417df9a742e6
Author: D G Turner <digitall@scummvm.org>
Date:   Sun Dec 15 22:22:20 2019 +0000

    AUDIO: Fix Compilation Against Fluidsynth v2.1+
    
    This is as reported by eriktorbjorn.

diff --git a/audio/softsynth/fluidsynth.cpp b/audio/softsynth/fluidsynth.cpp
index 4034b2ffc3..33a74a1524 100644
--- a/audio/softsynth/fluidsynth.cpp
+++ b/audio/softsynth/fluidsynth.cpp
@@ -35,7 +35,12 @@
 #include "backends/platform/ios7/ios7_common.h"
 #endif
 
+// Fluidsynth v2.1+ uses printf in one of it's headers, so this is
+// needed to allow compilation, as reported by eriktorbjorn on 20191215
+// This is in include/fluidsynth/log.h around line 82
+#define FORBIDDEN_SYMBOL_EXCEPTION_printf
 #include <fluidsynth.h>
+#undef FORBIDDEN_SYMBOL_EXCEPTION_printf
 
 class MidiDriver_FluidSynth : public MidiDriver_Emulated {
 private:
